build/push-official-release.sh

```diff
@@ -21,11 +21,12 @@ set -o nounset
 set -o pipefail
 
 if [ "$#" -ne 1 ]; then
```

Comment:
Except you handle the single-arg case, too.  So I guess `< 1 || > 2?  Also you should make it clear in Usage that the second argument is optional.